app/services/qa/linked_data/mapper/search_results_mapper_service.rb

```diff
@@ -33,13 +50,15 @@ class << self
           #      :altlabel=>[],
           #      :sort=>[#<RDF::Literal:0x3fcff54b4c18("2")>]}
           #   ]
-          def map_values(graph:, predicate_map:, sort_key:, preferred_language: nil, context_map: nil)
+          def map_values(graph:, prefixes: {}, ldpath_map: nil, predicate_map: nil, sort_key:, preferred_language: nil, context_map: nil) # rubocop:disable Metrics/ParameterLists
             search_matches = []
             graph.subjects.each do |subject|
               next if subject.anonymous? # skip blank nodes
-              values = graph_mapper_service.map_values(graph: graph, predicate_map: predicate_map, subject_uri: subject) do |value_map|
-                map_context(graph, sort_key, context_map, value_map, subject)
-              end
+              values = if ldpath_map.present?
+                         map_values_with_ldpath_map(graph: graph, ldpath_map: ldpath_map, prefixes: prefixes, subject_uri: subject, sort_key: sort_key, context_map: context_map)
+                       else
+                         map_values_with_predicate_map(graph: graph, predicate_map: predicate_map, subject_uri: subject, sort_key: sort_key, context_map: context_map)
+                       end
```

Comment:
Selects between processing using ldpath or predicates depending on which is defined in the search config.  Predicates continue to be supported for backward compatibility, but their use is deprecated.